contracts/protocol/integration/exchange/ZeroExApiAdapter.sol

```diff
@@ -152,6 +156,29 @@ contract ZeroExApiAdapter {
                 inputToken = fillData.tokens[0];
                 outputToken = fillData.tokens[fillData.tokens.length - 1];
                 inputTokenAmount = fillData.sellAmount;
+            } else if (selector == 0x6af479b2) {
+                // sellTokenForTokenToUniswapV3()
+                bytes memory encodedPath;
+                (encodedPath, inputTokenAmount, minOutputTokenAmount, recipient) =
+                    abi.decode(_data[4:], (bytes, uint256, uint256, address));
+                supportsRecipient = true;
+                (inputToken, outputToken) = _decodeTokensFromUniswapV3EncodedPath(encodedPath);
+            } else if (selector == 0x803ba26d) {
+                // sellTokenForEthToUniswapV3()
+                bytes memory encodedPath;
+                (encodedPath, inputTokenAmount, minOutputTokenAmount, recipient) =
+                    abi.decode(_data[4:], (bytes, uint256, uint256, address));
+                supportsRecipient = true;
+                (inputToken, outputToken) = _decodeTokensFromUniswapV3EncodedPath(encodedPath);
+            } else if (selector == 0x3598d8ab) {
+                // sellEthForTokenToUniswapV3()
+                // TODO(kimpers): is this correct?
+                inputTokenAmount = 0;
```

Comment:
Should this be set to 0 or do we just skip the inputAmount require when trading ETH?